101-mobile-app-create/azuredeploy.json

```diff
@@ -36,7 +36,12 @@
       "defaultValue": "Basic",
       "metadata": {
         "description": "The type of database to create. The available options are: Web, Business, Basic, Standard, and Premium."
-      }
+      },
```

Comment:
this list doesn't match the description - can we just move this to a variable and pick one?